tests/intrinsic/uart.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <stdint.h>
+
+struct uartlite_regs {
+    volatile unsigned int rx_fifo;
+    volatile unsigned int tx_fifo;
+    volatile unsigned int status;
+    volatile unsigned int control;
+};
+
+struct uartlite_regs *const ttyUL0 = (struct uartlite_regs *)0x60000000;
```

Comment:
`ttyUL0` should be more appropriate to be volatile than `struct uartlite_regs` members.